src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -6,17 +6,17 @@
      * С помощью тернарного оператора написать функцию, которая будет считать исход матча.
      * Метод будет принимать 4 параметра и возвращать целое число.
      * 4 параметра метода - результат матча и то, какую ставку сделал пользователь.
-     *
-     * Пример: 0,2,1,1. Что значит: команды сыграли в счёт 0:2, а пользователь ставил 1:1.
-     *
+     * <p>Пример: 0,2,1,1. Что значит: команды сыграли в счёт 0:2, а пользователь ставил 1:1.</p>
      * Задача:
-     *         если пользователь угадал 100% результат - возвращать 2
-     *         если угадал, что выиграла какая-либо команда
-     *             (пр. сыграли 1:2, а ставил 0:3 - то пользователь у выиграше)
-     *             то возвращать 1
-     *         если не угадал ничего - возвращать 0.
+     * если пользователь угадал 100% результат - возвращать 2
+     * если угадал, что выиграла какая-либо команда
+     * (пр. сыграли 1:2, а ставил 0:3 - то пользователь у выиграше)
+     * то возвращать 1
+     * если не угадал ничего - возвращать 0.
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1) && (result2 == bet2) ? 2 : ((result1 > result2) && (bet1 > bet2))
```

Comment:
```suggestion
        return result1 == bet1 && result2 == bet2 ? 2 : (result1 > result2 && bet1 > bet2)
```
Можна обійтися без зайвих ( i )